tests/wpt/web-platform-tests/fullscreen/reftests/fullscreen-baseline.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<head>
+  <meta charset=utf-8>
+  <title>Baseline fullscreen</title>
+  <link rel=match href=fullscreen-baseline-ref.html>
+  <style>
+    div { background: red; }
+    #it { background: green; }
+  </style>
+</head>
+<body>
+  <div>-1</div>
+  <div id="it">0</div>
+  <div>1</div>
+</body>
+<script>
+  document.getElementById("it").requestFullscreen();
```

Comment:
This won't work in non-servo browsers which require user activation to enter fullscreen.

Reply:
True, but we'd need other browsers to add support for yesterday which request fullscreen but expect to reuse the current window. Such a testing pref could also allow fullscreen without user engagement.